src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -32,6 +32,7 @@
     <Compile Include="Implementation\Preview\ReferenceChange.ProjectReferenceChange.cs" />
```

Comment:
Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declation.